src/core/wrap.js

```diff
@@ -280,6 +286,43 @@ Object.assign(wrap, {
             });
         };
     },
+    cleanDocs: function(info){
+        if(info.docs) for(const key in info.docs){
```

Comment:
once the documentation format is standardized this should only clean the specified keys, not all string values